benchmarks/README.md

```diff
@@ -0,0 +1,65 @@
+# FastTrack ML Benchmark Suite
```

Comment:
Could you please ensure the project is spelled FastTrackML everywhere?

Reply:
Alright @jgiannuzzi  would make that change now